tests/functional/aws-node-sdk/test/versioning/multiObjectDelete.js

```diff
@@ -140,7 +140,7 @@ testing('Multi-Object Versioning Delete Success', function success() {
     });
 });
 
-testing('Multi-Object Versioning Delete - deleting delete marker',
+describe('Multi-Object Versioning Delete - deleting delete marker',
```

Comment:
if you change 'describe' to 'testing' these tests will probably not pass the ci until https://github.com/scality/S3/pull/690 is merged because versioning is not implemented for bucketfile yet